src/bitcoin-cli.cpp

```diff
@@ -294,6 +294,8 @@ int CommandLineRPC(int argc, char *argv[])
     catch (const boost::thread_interrupted&) {
```

Comment:
No, see "future changes" in the OP. There are a bunch of these to get rid of. I guess it doesn't make sense to do that later, I'll add it here.